test/inductor/test_flex_attention.py

```diff
@@ -2507,6 +2507,28 @@ def mask_mod(b, h, q, kv):
         out = func(query, key, value, block_mask=block_mask)
         out.sum().backward()
 
+    @supported_platform
+    def test_strided_backwards(self):
+        shape = (1, 2, 4096, 64)
+        Q = torch.randn(shape, requires_grad=True, device="cuda", dtype=torch.bfloat16)
+        K = torch.randn(shape, requires_grad=True, device="cuda", dtype=torch.bfloat16)
+        V = torch.randn(shape, requires_grad=True, device="cuda", dtype=torch.bfloat16)
```

Comment:
```suggestion
        Q = torch.randn(shape, requires_grad=True, device="cuda")
        K = torch.randn(shape, requires_grad=True, device="cuda")
        V = torch.randn(shape, requires_grad=True, device="cuda")
```

https://github.com/pytorch/pytorch/pull/146563/files#r1945630309